src/CapabilityService.WebApi/Infrastructure/Messaging/IRepository.cs

```diff
@@ -6,5 +6,7 @@ namespace DFDS.CapabilityService.WebApi.Infrastructure.Messaging
     public interface IRepository<T> where T:class
     {
         Task Add(IEnumerable<T> obj);
+
+        Task<IEnumerable<T>> GetAll();
```

Comment:
In general I like using Get(Predication<T> filter = default), rather then GetAll() in repo interfaces, to avoid tons of GetByXYZ style methods, but thats more of a suggestion then an actual comment. :)

Reply:
I am making a note of looking into that